src/main/java/io/zksync/signer/SigningUtils.java

```diff
@@ -15,7 +16,7 @@
 
 public class SigningUtils {
 
-    private static final int MAX_NUMBER_OF_TOKENS = 128;
+    private static final long MAX_NUMBER_OF_TOKENS = Long.MAX_VALUE;
```

Comment:
It should be `unsigned 2^32`

Reply:
Unfortunately Java has no any unsigned type thus actually uses greater types to cover set of unsigned values. As alternative I can add invariant checking within convertion into bytes